pennylane/ops/qubit/parametric_ops.py

```diff
@@ -471,23 +547,49 @@ class MultiRZ(Operation):
     def num_params(self):
         return 1
 
-    @classmethod
-    def _matrix(cls, theta, n):
-        """Matrix representation of a MultiRZ gate.
+    @staticmethod
+    def compute_matrix(theta, n_wires):
```

Comment:
This is another case where we overwrite, because info on the wires needs to be provided

Reply:
@mariaschuld should this be updated to use `hyperaparams`?

Reply:
Yes, this is one of the examples. Let me do it and see how far I get!